codebuild/spec/buildspec_al2023_ktls.yml

```diff
@@ -11,8 +11,10 @@ phases:
       - yum update -y; yum upgrade -y
   pre_build:
     commands:
-      # Nix is installed, but intentionally not setup for root, fix that
-      - cp -aR /home/nix/.nix-profile ~/; chown -R root /root/.nix-profile; export PATH=$HOME/.nix-profile/bin:$PATH
+      # Install nix
+      - sh <(curl -L https://nixos.org/nix/install) --no-daemon
```

Comment:
Why is this necessary as opposed to just vanilla curl?

Reply:
The curl command inside the parentheses gets the installation script. However, in order to actually run the script you have to pipe it to sh. I'm basically just following the nix install guide https://nix.dev/manual/nix/2.18/installation/installation.

Reply:
This would probably also work:
`curl -L https://nixos.org/nix/install | sh`

Reply:
> This would probably also work: `curl -L https://nixos.org/nix/install | sh`

We want that `--no-daemon` flag.